### DISCUSSION TITLE
SO 2.3.130 - Suricata stopped to create alarms

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#8111] [Unsupported Versions] [answered]
Hello!

Last Saturday Suricata stopped to create alarms and I have no clue why. I can't find any signs of errors. so-status looks fine on the manager and forward node and also the suricata.log shows no errors or warnings. Data is comming in as usal (zeek, syslogs and the other regular stuff ...). The only alarms I see in SOC are generated by Ossec/Wazuh. If I search for Suricata alarms in Elasticsearch, I find nothing since June 11, 1am. Why?

Cheers, Ben

**Comment by xfaith** [comment]
First things first when loosing Alerts I would validate the HOME_NET that your SO is set to.  Has anything in your network changed?  DHCP?
Have you made any minion/global salt changes recently?

**Reply by ben-sec** [reply]
When I left the office on June 10 everything worked fine and I didn't change a thing that day (especially not the HOME_NET. I don't use DHCP for my nodes and as I mentioned before, everything else is working fine. BTW: I did restart Suricata and also a complete reboot of my forward node. No errors, but also no alarms.

**Comment by dougburks** [comment]
Have you checked the Troubleshooting Alerts section of our documentation?
https://docs.securityonion.net/en/2.3/suricata.html#troubleshooting-alerts

**Reply by ben-sec** [reply]
I already checked all of them, but I did not setup a new import node yet. I have a lot of BPF rules in global.sls and noticed in suricata.log, that I have entries starting with
`14/6/2022 -- 07:48:13 - <Info> - Using BPF ...` , but the entries don't show the complete set of BPF rules that I have configured. Get the entries just truncated in the log? Again, Suricata is running and I see the following in stats.log:

![image](https://user-images.githubusercontent.com/81571847/173572968-4ab4a495-14e6-46df-899f-b621c3b94aad.png)

**Reply by ben-sec** [reply]
I have removed all NIDS BPF rules, but there are still no Suricata alarms comming in (I just wanted to make sure that there is no typo somewhere ..). :-(

**Comment by ben-sec** [comment]
I know, this is highly appreciated best effort community support, but I need a solution, since we are sitting here without alarms since Saturday. Just let me know If buying Premium Support would speed up things. Thanks again for your hard work! Big Fan!

**Reply by dougburks** [reply]
Yes, Premium Support includes private support with priority response:
https://securityonionsolutions.com/support/

**Reply by ben-sec** [reply]
OK, I will go and get some approvals and wait for your offer (I filled out your online form ...). In the meanwhile: Any more ideas what I can try?

**Comment by ben-sec** [comment, marked as answer]
OK, found the issue: I reached the max number of ES shards and Logstash was not able to index IDS alerts to ES. I decreased the number of days before an index gets closed in global.sls.
